tests/integration/test_fixtures.py

```diff
@@ -36,7 +36,7 @@ def finalizer():
             cassette_name = ('tests.integration.test_fixtures.'  # Module name
                              'TestPyTestParametrizedFixtures.'  # Class name
                              'test_pytest_fixture'  # Test function name
-                             '[https:-httpbin.org-get]'  # Parameter
+                             '[https---httpbin.org-get]'  # Parameter
```

Comment:
Because I also propose to remove the `+` at the end of the `_sanitize` regexp, which has the effect of keeping the same number of replaced characters, instead of mapping a bunch of them into a single `-`. For instance, `aaa//bbb` would have been mapped to `aaa-bbb` before, while I now propose to map it to `aaa--bbb`, which I find more explicit.